src/query-mapper/mapper/value.mapper.ts

```diff
@@ -0,0 +1,18 @@
+import { AbstractParameterMapper } from "./parameter-mapper.interface"
+
+// todo type check?!
+export class ValueMapper extends AbstractParameterMapper {
+    doMap(parameter: any) {
+        if (parameter == null) {
+            return null
+        }
+        if (typeof parameter == 'object' || Array.isArray(parameter)) {
```

Comment:
second condition isn't needed, I think

Reply:
this is done to accommodate that req.query params sometime comes as array or as object. didn't understand the logic. but this accommodates both cases.